mitmproxy/tools/console/options.py

```diff
@@ -45,7 +45,10 @@ def get_widget(self):
         if self.opt.typespec == bool:
             displayval = "true" if val else "false"
         elif not val:
-            displayval = ""
+            if self.opt.typespec == float:
```

Comment:
Could you explain this change? In particular, why do we need to treat floats and ints differently here?

Reply:
@mhils what I noticed was that int and float options with a default value of `0` or `0.0` displayed the empty string. I think that the actual `0` or `0.0` should be displayed but since I was working only on floats I'd decided to leave `int` as is and instead handle the case when its a float to display the string value of it.

If you're ok with it I can make it so that the `0` and `0.0` is displayed for both int and float respectively when their values are `0`

Reply:
That makes sense, thanks. Let's change that condition in line 47 to `elif val in (None, ""):`, then we have proper treatment for everything else.